lib/node_modules/@stdlib/array/complex64/README.md

```diff
@@ -856,6 +856,72 @@ var count = context.count;
 // returns 3
 ```
 
+<a name="method-filter"></a>
+
+#### Complex64Array.prototype.filter( predicate\[, thisArg])
+
+Returns the array containing the elements of an array which pass a test implemented by a predicate function.
+
+```javascript
+var realf = require( '@stdlib/complex/realf' );
+var imagf = require( '@stdlib/complex/imagf' );
+
+function predicate( v ) {
+    return ( realf( v ) === imagf( v ) );
+}
+
+var arr = new Complex64Array( 3 );
+
+// Set the first three elements:
+arr.set( [ 1.0, -1.0 ], 0 );
+arr.set( [ 2.0, 2.0 ], 1 );
+arr.set( [ 3.0, -3.0 ], 2 );
+
+var out = arr.filter( predicate );
+// returns <Complex64Array>
+
+var len = out.length;
+// returns 1
```

Comment:
@Jaysukh-409 I suggest extending this example to actually show what the first, and only, element is for the returned array. You can use `get`, followed by `realf` and `imagf`.